css/css-grid/parsing/display-invalid.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Grid Layout Module Level 1: parsing display with invalid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#propdef-display">
+<meta name="assert" content="display supports the new values 'grid | inline-grid'.">
```

Comment:
This is not being checked in this test we should reword this sentence or move these checks to the other test (like I'm suggesting in the next comment).